Crashing on boot with pre-release

[Log file](https://gist.github.com/OsoaGitHub/26de2c34e4472f7f008f14a2657ec6f0)

I'm gonna go through my mods, uninstalling a batch at a time to see which one is the culprit, shouldn't take too long.

<details>
<summary>Truncated log with Fabrication 1.3.5-pre1 and SGM</summary>
<pre>
[20:27:57] [pool-2-thread-1/WARN]: @Redirect conflict. Skipping fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen->@Redirect::drawDirt(Lnet/minecraft/class_766;FF)V with priority 1000, already redirected by slight-gui-modifications.mixins.json:MixinTitleScreen->@Redirect::thing(Lnet/minecraft/class_766;FF)V with priority 1000
[20:27:57] [pool-2-thread-1/INFO]: Instancing error handler class com.unascribed.fabrication.support.MixinErrorHandler_THIS_ERROR_HANDLER_IS_FOR_SOFT_FAILURE_IN_FABRICATION_ITSELF_AND_DOES_NOT_IMPLY_FABRICATION_IS_RESPONSIBLE_FOR_THE_BELOW_ERROR
[20:27:57] [pool-2-thread-1/FATAL]: Mixin apply failed fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen -> net.minecraft.class_442: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Critical injection failure: Redirector drawDirt(Lnet/minecraft/class_766;FF)V in fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap fabrication-refmap.json [INJECT Applicator Phase -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen -> Apply Injections ->  -> PostInject -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen->@Redirect::drawDirt(Lnet/minecraft/class_766;FF)V]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: Redirector drawDirt(Lnet/minecraft/class_766;FF)V in fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap fabrication-refmap.json [INJECT Applicator Phase -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen -> Apply Injections ->  -> PostInject -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen->@Redirect::drawDirt(Lnet/minecraft/class_766;FF)V]
	at com.unascribed.fabrication.support.injection.Failsoft.postInject(Failsoft.java:98) ~[fabrication-1.3.5-pre1.jar:?]
	at com.unascribed.fabrication.support.injection.FailsoftRedirectInjectionInfo.postInject(FailsoftRedirectInjectionInfo.java:21) ~[fabrication-1.3.5-pre1.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1291) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23) ~[fabric-loader-0.11.2.jar:0.9.2+mixin.0.8.2]
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157) ~[fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150) ~[fabric-loader-0.11.2.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:522) ~[?:?]
	at com.unascribed.fabrication.Analytics.addClientData(Analytics.java:211) ~[fabrication-1.3.5-pre1.jar:?]
	at com.unascribed.fabrication.Analytics.lambda$submit$0(Analytics.java:104) ~[fabrication-1.3.5-pre1.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
[20:27:57] [main/WARN]: @Redirect conflict. Skipping fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen->@Redirect::drawDirt(Lnet/minecraft/class_766;FF)V with priority 1000, already redirected by slight-gui-modifications.mixins.json:MixinTitleScreen->@Redirect::thing(Lnet/minecraft/class_766;FF)V with priority 1000
[20:27:57] [main/INFO]: Instancing error handler class com.unascribed.fabrication.support.MixinErrorHandler_THIS_ERROR_HANDLER_IS_FOR_SOFT_FAILURE_IN_FABRICATION_ITSELF_AND_DOES_NOT_IMPLY_FABRICATION_IS_RESPONSIBLE_FOR_THE_BELOW_ERROR
[20:27:57] [main/FATAL]: Mixin apply failed fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen -> net.minecraft.class_442: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Critical injection failure: Redirector drawDirt(Lnet/minecraft/class_766;FF)V in fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap fabrication-refmap.json [INJECT Applicator Phase -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen -> Apply Injections ->  -> PostInject -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen->@Redirect::drawDirt(Lnet/minecraft/class_766;FF)V]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: Redirector drawDirt(Lnet/minecraft/class_766;FF)V in fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap fabrication-refmap.json [INJECT Applicator Phase -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen -> Apply Injections ->  -> PostInject -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen->@Redirect::drawDirt(Lnet/minecraft/class_766;FF)V]
	at com.unascribed.fabrication.support.injection.Failsoft.postInject(Failsoft.java:98) ~[fabrication-1.3.5-pre1.jar:?]
	at com.unascribed.fabrication.support.injection.FailsoftRedirectInjectionInfo.postInject(FailsoftRedirectInjectionInfo.java:21) ~[fabrication-1.3.5-pre1.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1291) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23) [fabric-loader-0.11.2.jar:0.9.2+mixin.0.8.2]
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150) [fabric-loader-0.11.2.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:522) [?:?]
	at net.minecraft.client.main.Main.main(Main.java:177) [intermediary-minecraft-1.16.5-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28) [fabric-loader-0.11.2.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
[20:27:57] [main/WARN]: @Redirect conflict. Skipping fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen->@Redirect::drawDirt(Lnet/minecraft/class_766;FF)V with priority 1000, already redirected by slight-gui-modifications.mixins.json:MixinTitleScreen->@Redirect::thing(Lnet/minecraft/class_766;FF)V with priority 1000
[20:27:57] [main/INFO]: Instancing error handler class com.unascribed.fabrication.support.MixinErrorHandler_THIS_ERROR_HANDLER_IS_FOR_SOFT_FAILURE_IN_FABRICATION_ITSELF_AND_DOES_NOT_IMPLY_FABRICATION_IS_RESPONSIBLE_FOR_THE_BELOW_ERROR
[20:27:57] [main/FATAL]: Mixin apply failed fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen -> net.minecraft.class_442: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Critical injection failure: Redirector drawDirt(Lnet/minecraft/class_766;FF)V in fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap fabrication-refmap.json [INJECT Applicator Phase -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen -> Apply Injections ->  -> PostInject -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen->@Redirect::drawDirt(Lnet/minecraft/class_766;FF)V]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: Redirector drawDirt(Lnet/minecraft/class_766;FF)V in fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap fabrication-refmap.json [INJECT Applicator Phase -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen -> Apply Injections ->  -> PostInject -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen->@Redirect::drawDirt(Lnet/minecraft/class_766;FF)V]
	at com.unascribed.fabrication.support.injection.Failsoft.postInject(Failsoft.java:98) ~[fabrication-1.3.5-pre1.jar:?]
	at com.unascribed.fabrication.support.injection.FailsoftRedirectInjectionInfo.postInject(FailsoftRedirectInjectionInfo.java:21) ~[fabrication-1.3.5-pre1.jar:?]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1291) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23) [fabric-loader-0.11.2.jar:0.9.2+mixin.0.8.2]
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150) [fabric-loader-0.11.2.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:522) [?:?]
	at net.minecraft.client.main.Main.main(Main.java:185) [intermediary-minecraft-1.16.5-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28) [fabric-loader-0.11.2.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
[20:27:57] [main/INFO]: [STDERR]: Failed to start Minecraft:
[20:27:57] [main/INFO]: [STDERR]: java.lang.reflect.InvocationTargetException
[20:27:57] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[20:27:57] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[20:27:57] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[20:27:57] [main/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[20:27:57] [main/INFO]: [STDERR]: 	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
[20:27:57] [main/INFO]: [STDERR]: 	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
[20:27:57] [main/INFO]: [STDERR]: 	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
[20:27:57] [main/INFO]: [STDERR]: 	at org.multimc.EntryPoint.main(EntryPoint.java:34)
[20:27:58] [main/INFO]: [STDERR]: Caused by: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
[20:27:58] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:228)
[20:27:58] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
[20:27:58] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
[20:27:58] [main/INFO]: [STDERR]: 	... 8 more
[20:27:58] [main/INFO]: [STDERR]: Caused by: java.lang.reflect.InvocationTargetException
[20:27:58] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[20:27:58] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[20:27:58] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[20:27:58] [main/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[20:27:58] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
[20:27:58] [main/INFO]: [STDERR]: 	... 10 more
[20:27:58] [main/INFO]: [STDERR]: Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208)
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178)
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23)
[20:27:58] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157)
[20:27:58] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150)
[20:27:58] [main/INFO]: [STDERR]: 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
[20:27:58] [main/INFO]: [STDERR]: 	at net.minecraft.client.main.Main.main(Main.java:185)
[20:27:58] [main/INFO]: [STDERR]: 	... 15 more
[20:27:58] [main/INFO]: [STDERR]: Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen] from phase [DEFAULT] in config [fabrication.mixins.json] FAILED during APPLY
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:642)
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:594)
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:356)
[20:27:58] [main/INFO]: [STDERR]: 	... 22 more
[20:27:58] [main/INFO]: [STDERR]: Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: Redirector drawDirt(Lnet/minecraft/class_766;FF)V in fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap fabrication-refmap.json [INJECT Applicator Phase -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen -> Apply Injections ->  -> PostInject -> fabrication.mixins.json:i_woina.dirt_screen.MixinTitleScreen->@Redirect::drawDirt(Lnet/minecraft/class_766;FF)V]
[20:27:58] [main/INFO]: [STDERR]: 	at com.unascribed.fabrication.support.injection.Failsoft.postInject(Failsoft.java:98)
[20:27:58] [main/INFO]: [STDERR]: 	at com.unascribed.fabrication.support.injection.FailsoftRedirectInjectionInfo.postInject(FailsoftRedirectInjectionInfo.java:21)
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1291)
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042)
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395)
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320)
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345)
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569)
[20:27:58] [main/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351)
[20:27:58] [main/INFO]: [STDERR]: 	... 22 more
[20:27:58] [main/INFO]: [STDOUT]: Exiting with -1
</pre>
</details>

--- a/src/main/java/com/unascribed/fabrication/mixin/i_woina/dirt_screen/MixinTitleScreen.java
+++ b/src/main/java/com/unascribed/fabrication/mixin/i_woina/dirt_screen/MixinTitleScreen.java
@@ -27,15 +27,8 @@ public class MixinTitleScreen extends Screen {
         super(title);
     }
 
-    @Shadow
-    @Final
-    boolean doBackgroundFade;
-
-    @Shadow
-    long backgroundFadeStart;
-
     @Inject(method = "render", at = @At(value="INVOKE", target="Lnet/minecraft/client/gui/RotatingCubeMapRenderer;render(FF)V", shift = At.Shift.AFTER))
-    public void drawDirt(MatrixStack matrices, int mouseX, int mouseY, float delta, CallbackInfo ci) {
+    public void drawDirt(CallbackInfo ci) {
         if (MixinConfigPlugin.isEnabled("*.dirt_screen")) {
             renderBackgroundTexture(0);
         }

--- a/src/main/java/com/unascribed/fabrication/mixin/i_woina/dirt_screen/MixinTitleScreen.java
+++ b/src/main/java/com/unascribed/fabrication/mixin/i_woina/dirt_screen/MixinTitleScreen.java
@@ -34,14 +34,10 @@ public class MixinTitleScreen extends Screen {
     @Shadow
     long backgroundFadeStart;
 
-    @Redirect(method = "render", at = @At(value="INVOKE", target="Lnet/minecraft/client/gui/RotatingCubeMapRenderer;render(FF)V"))
-    public void drawDirt(RotatingCubeMapRenderer rotatingCubeMapRenderer, float delta, float alpha) {
+    @Inject(method = "render", at = @At(value="INVOKE", target="Lnet/minecraft/client/gui/RotatingCubeMapRenderer;render(FF)V", shift = At.Shift.AFTER))
+    public void drawDirt(MatrixStack matrices, int mouseX, int mouseY, float delta, CallbackInfo ci) {
         if (MixinConfigPlugin.isEnabled("*.dirt_screen")) {
             renderBackgroundTexture(0);
-        }
-        else {
-            float f = doBackgroundFade ? (float)(Util.getMeasuringTimeMs() - backgroundFadeStart) / 1000.0F : 1.0F;
-            rotatingCubeMapRenderer.render(delta, MathHelper.clamp(f,0.0F, 1.0F));
         }
     }
 }

